tools/experiments/experiments.js

```diff
@@ -309,6 +309,11 @@ const EXPERIMENTS = [
     spec: 'https://github.com/ampproject/amphtml/issues/14740',
     cleanupissue: 'https://github.com/ampproject/amphtml/issues/16075',
   },
+  {
+    id: 'amp-img-blurry-placeholder',
+    name: 'Enables a blurry image placeholder as an amp-img loads',
+    spec: 'https://github.com/ampproject/amphtml/issues/15146',
```

Comment:
File a new issue for cleaning up this experiment and add it to `cleanupissue` key here.

Reply:
Filed an issue, but I don't have access to add labels. Sorry if I'm wrong, but if you can add it/tell me how that would be appreciated.